magefile.go

```diff
@@ -179,7 +181,7 @@ func Test() error {
 			tst := goRun.Test
 			bin := path.Join(home, path.Base(pkg))
 			useBinary := fs.ExistsFile(bin) && testUseBin
-			fmt.Println("Run test for package '%v' (binary: %v)", pkg, useBinary)
+			fmt.Printf("Run test for package '%v' (binary: %v)\n", pkg, useBinary)
```

Comment:
Nit: why not use `%s` for substituting `pkg` which is a string?

Reply:
What is the advantage of using `%s` over `%v`? When passing primitive types the fmt package uses the same code path for `%v` and `%s`.

Reply:
No advantage. I guess it's just a personal preference. :)